src/tests/FunctionalTests/WebAssembly/Browser/RuntimeConfig/WebAssembly.Browser.RuntimeConfig.Test.csproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <MonoForceInterpreter>true</MonoForceInterpreter>
+    <RunAOTCompilation>false</RunAOTCompilation>
+    <PublishTrimmed>false</PublishTrimmed>
+    <TestRuntime>true</TestRuntime>
+    <Scenario>WasmTestOnBrowser</Scenario>
+    <ExpectedExitCode>42</ExpectedExitCode>
+    <WasmMainJSPath>runtime.js</WasmMainJSPath>
+    <EnableDefaultItems>false</EnableDefaultItems>
```

Comment:
this shouldn't be needed

Reply:
All of them? To my understanding, line 7-9 should be needed. But I could be wrong.

Reply:
Setting `EnableDefaultItems=false` is useful when you want to explicitly control which source files to include, which resx files etc. But IIUC, that's not needed here.

So, if we don't set this, then even line 11(`<Compile Include="Program.cs" />`) won't be needed.

Reply:
`<Compile Include="Program.cs" />` is indeed needed, otherwise, I hit this error
```
CSC : error CS5001: Program does not contain a static 'Main' method suitable for an entry point [/Users/fanyang/Documents/work/dotnet_wasm/runtime/src/tests/FunctionalTests/WebAssembly/Browser/RuntimeConfig/WebAssembly.Browser.RuntimeConfig.Test.csproj]
```